package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "scratch-js",
+  "version": "0.1.0",
+  "description": "Library for making Scratch-like projects with JavaScript.",
+  "main": "scratch-js/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint 'scratch-js/**'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/PullJosh/scratch-js.git"
+  },
+  "author": "Josh Pullen",
+  "license": "ISC",
```

Comment:
Let's put `"LGPL-3.0-or-later"` here unless someone has a better recommendation.

Reply:
Actually, maybe `"MIT"` is the move.

Reply:
I believe the LGPL permits usage of linked libraries, but if you e.g. include `scratch-js` in, say, your game project with a bundler, that's a gray area.

Reply:
Does spending too long thinking about licenses count as [bikeshedding](https://en.wiktionary.org/wiki/bikeshedding)? :P